userguide/system.rst

```diff
@@ -2288,21 +2288,51 @@ Updating an HA System
 
 If the %brand% array has been configured for High Availability
 (HA), the update process must be started on the active |ctrlr-term|.
-Once the update is complete, the standby |ctrlr-term| will automatically
-reboot. Wait for it to come back up by monitoring the remote console or
-the |web-ui| of the standby |ctrlr-term|.
+The update process is to download and synchronize the update files
```

Comment:
"process is to" is ambiguous.  How about just a colon?  "The update process is: download and..."